pkgs/by-name/lw/lwk/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, udev }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lwk";
+  version = "0.2.0";
+  cargoBuildFlags = "--bin lwk_cli";
+
+  src = fetchFromGitHub {
+    owner = "Blockstream";
+    repo = pname;
+    rev = "f1e1473b20f0699c9419986094e094786cb30be4";
+    hash = "sha256-+MyD2usM9IfJmS5sP+hsYGhqTquUj9RQUmEt50rMyOs=";
+  };
+
+  cargoLock = { lockFile = ./Cargo.lock; };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  doCheck = false;
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ udev ];
+
+  meta = with lib; {
+    description =
+      "Liquid Wallet Kit (LWK) is a CLI program for Liquid descriptor wallets, with support for watch-only, multisig, hardware signers, and asset management.";
```

Comment:
Please keep the description short and exclude the name of the program and the trailing period; `meta.longDescription` can be longer.